lib/kubernetes-deploy/kubernetes_resource/horizontal_pod_autoscaler.rb

```diff
@@ -2,16 +2,17 @@
 module KubernetesDeploy
   class HorizontalPodAutoscaler < KubernetesResource
     TIMEOUT = 3.minutes
-    RECOVERABLE_CONDITIONS = %w(ScalingDisabled FailedGet)
+    RECOVERABLE_CONDITION_PREFIX = "FailedGet"
 
     def deploy_succeeded?
-      scaling_active_condition["status"] == "True"
+      scaling_active_condition["status"] == "True" || scaling_disabled?
```

Comment:
To me this is analogous to the situation where a service exists and is properly configured but has no back ends because the related deployment is scaled to zero. In both situations, the dependent resource is doing the right thing, but the right thing happens to be "disable". We do also already expose the situation to the user via the status string. I'm hesitant to add a warning about something that is actually expected behavior.